### DISCUSSION TITLE
Make includeFile async

### DISCUSSION THREAD

**Discussion by origranot** [eta-dev/eta#91] [Q&A]
Hey,

When I try to include my menu partial before content the partial being renderd after the content.

**Comment by shadowtime2000** [comment]
I am a little confused by what you mean by this - I am finding it a little hard to match the title with what you are saying to figure out what you are asking. Can you clarify?

**Reply by origranot** [reply]
Nevermind, Thank you for your reply. Managed to do it with layouts!
Super easy, thank you.

**Reply by shadowtime2000** [reply]
Glad you were able to get it working!
